test/api/test_history_contents.py

```diff
@@ -1,11 +1,10 @@
-from base import api
 import json
 
-from .helpers import TestsDatasets
-from .helpers import LibraryPopulator
-from .helpers import DatasetCollectionPopulator
-from requests import delete
-from requests import put
+from requests import delete, put
+
+from base import api
+
+from .helpers import DatasetCollectionPopulator, LibraryPopulator, TestsDatasets
```

Comment:
I guess relative imports is the way to fix the unit tests huh? Or creating a galaxy_test module and reworking everything I guess.